autoload/ale/fixers/rubocop.vim

```diff
@@ -1,15 +1,18 @@
 call ale#Set('ruby_rubocop_options', '')
+call ale#Set('ruby_rubocop_auto_correct_all', '')
 call ale#Set('ruby_rubocop_executable', 'rubocop')
 
 function! ale#fixers#rubocop#GetCommand(buffer) abort
     let l:executable = ale#Var(a:buffer, 'ruby_rubocop_executable')
     let l:config = ale#path#FindNearestFile(a:buffer, '.rubocop.yml')
     let l:options = ale#Var(a:buffer, 'ruby_rubocop_options')
+    let l:auto_correct_all = ale#Var(a:buffer, 'ruby_rubocop_auto_correct_all')
 
     return ale#ruby#EscapeExecutable(l:executable, 'rubocop')
     \   . (!empty(l:config) ? ' --config ' . ale#Escape(l:config) : '')
     \   . (!empty(l:options) ? ' ' . l:options : '')
-    \   . ' --auto-correct --force-exclusion %t'
+    \   . (!empty(l:auto_correct_all) ? ' --auto-correct-all' : ' --auto-correct')
```

Comment:
```suggestion
    \   . (l:auto_correct_all ? ' --auto-correct-all' : ' --auto-correct')
```

Do this instead, and document this new option as a `Number`. Update the tests to check the command we run when it's set to `1`.